repo/packages/H/history/0/config.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "type": "object",
+  "properties": {
+    "mesos/master/url": {
```

Comment:
the defacto standard here looks like this: https://github.com/mesosphere/universe/blob/version-1.x/repo/packages/M/marathon/1/config.json#L179-L192